spec/lib/object_serializer_spec.rb

```diff
@@ -36,6 +40,17 @@
       expect(serializable_hash['meta']).to be_instance_of(Hash)
     end
 
+    it 'returns correct fieldset when serializer instantiated with fields option' do
```

Comment:
I think it makes sense to add a performance spec too. That would give us an idea of how this feature performs in fast_jsonapi compared to AMS.

Reply:
Ok, will do that